Defs/Things.xml

```diff
@@ -217,4 +217,67 @@
       </recipeUsers>
     </recipeMaker>
   </ThingDef>
+
+<!--Don't foret to update tex paths when we get the new textures-->
+  <ThingDef ParentName="BeddingBase">
+    <defName>SingleTribalBedding</defName>
+    <label>single tribal bedding set</label>
+    <description>A pile of leather blankets with a rudimentary pillow. Not very comfortable but much better than nothing and insulates the user while in bed.</description>
+    <graphicData>
+      <texPath>Things/Beddings/beddingSingle</texPath>
+      <graphicClass>Graphic_Single</graphicClass>
+    </graphicData>
+    <costStuffCount>55</costStuffCount>
+    <stuffCategories>
+      <li>Leathery</li>
+    </stuffCategories>
+    <tradeability>Sellable</tradeability>
+    <statBases>
+      <MaxHitPoints>100</MaxHitPoints>
+      <WorkToMake>400</WorkToMake>
+      <Mass>10</Mass>
+    </statBases>
+    <thingCategories>
+      <li>Beddings</li>
+    </thingCategories>
+    <recipeMaker>
+      <researchPrerequisite>CarpetMaking</researchPrerequisite>
```

Comment:
So if you're playing with VFEVikings and VFurniture Expanded you'll need to research simple furniture.

This was for the few people who are only playing with Soft Beds and Vikings. From looking at the defs in the viking mod there isn't a prerequisite research for the bed so that'd mean tribal beddings are availiable from the start of the game. I'm not sure how to set it to have no prerequisite, and still have the patch operation that moves it to the VFurniture research.

Reply:
Hmmm, I see. For a no-prerequisite thing just remove that line. To add it back, it would be a PatchOperationAdd triggered by a PatchOperationFindMod (or in a sequence that already detected the mod).